--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	deleted:    ISAAC_NOGIT/isaac_model/omniquad/new_omni.usd
	deleted:    ISAAC_NOGIT/isaac_model/omniquad/omni_prova.usd
	modified:   ISAAC_NOGIT/isaac_model/traquad/traquad.usd
	modified:   ISAAC_NOGIT/source/isaaclab/isaaclab/envs/mdp/commands/__pycache__/commands_cfg.cpython-311.pyc
	modified:   ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/omniquad.cpython-311.pyc
	modified:   ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_omni_cfg.cpython-311.pyc
	modified:   ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/__pycache__/rough_env_cfg.cpython-311.pyc
	modified:   ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc
	modified:   ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/velocity_env_traquad_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	ISAAC_NOGIT/isaac_model/traquad/configuration/
	ISAAC_NOGIT/logs/rsl_rl/omniquad_rough/2025-09-12_10-55-00/
	ISAAC_NOGIT/logs/rsl_rl/omniquad_rough/2025-09-12_11-52-13/
	ISAAC_NOGIT/logs/rsl_rl/omniquad_rough/2025-09-12_11-55-30/
	ISAAC_NOGIT/outputs/2025-09-12/10-55-00/
	ISAAC_NOGIT/outputs/2025-09-12/10-55-45/
	ISAAC_NOGIT/outputs/2025-09-12/11-32-57/
	ISAAC_NOGIT/outputs/2025-09-12/11-36-14/
	ISAAC_NOGIT/outputs/2025-09-12/11-39-06/
	ISAAC_NOGIT/outputs/2025-09-12/11-52-13/
	ISAAC_NOGIT/outputs/2025-09-12/11-55-30/
	ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/traquad.cpython-311.pyc
	ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_traquad_cfg.cpython-311.pyc

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/ISAAC_NOGIT/isaac_model/omniquad/new_omni.usd b/ISAAC_NOGIT/isaac_model/omniquad/new_omni.usd
deleted file mode 100644
index 6827609..0000000
Binary files a/ISAAC_NOGIT/isaac_model/omniquad/new_omni.usd and /dev/null differ
diff --git a/ISAAC_NOGIT/isaac_model/omniquad/omni_prova.usd b/ISAAC_NOGIT/isaac_model/omniquad/omni_prova.usd
deleted file mode 100644
index 939a69d..0000000
Binary files a/ISAAC_NOGIT/isaac_model/omniquad/omni_prova.usd and /dev/null differ
diff --git a/ISAAC_NOGIT/isaac_model/traquad/traquad.usd b/ISAAC_NOGIT/isaac_model/traquad/traquad.usd
index 121c448..127a4de 100644
Binary files a/ISAAC_NOGIT/isaac_model/traquad/traquad.usd and b/ISAAC_NOGIT/isaac_model/traquad/traquad.usd differ
diff --git a/ISAAC_NOGIT/source/isaaclab/isaaclab/envs/mdp/commands/__pycache__/commands_cfg.cpython-311.pyc b/ISAAC_NOGIT/source/isaaclab/isaaclab/envs/mdp/commands/__pycache__/commands_cfg.cpython-311.pyc
index 2989066..803c338 100644
Binary files a/ISAAC_NOGIT/source/isaaclab/isaaclab/envs/mdp/commands/__pycache__/commands_cfg.cpython-311.pyc and b/ISAAC_NOGIT/source/isaaclab/isaaclab/envs/mdp/commands/__pycache__/commands_cfg.cpython-311.pyc differ
diff --git a/ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/omniquad.cpython-311.pyc b/ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/omniquad.cpython-311.pyc
index 150d1ce..3e16329 100644
Binary files a/ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/omniquad.cpython-311.pyc and b/ISAAC_NOGIT/source/isaaclab_assets/isaaclab_assets/robots/__pycache__/omniquad.cpython-311.pyc differ
diff --git a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_omni_cfg.cpython-311.pyc b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_omni_cfg.cpython-311.pyc
index 27b8926..10be50c 100644
Binary files a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_omni_cfg.cpython-311.pyc and b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/__pycache__/velocity_env_omni_cfg.cpython-311.pyc differ
diff --git a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/__pycache__/rough_env_cfg.cpython-311.pyc b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/__pycache__/rough_env_cfg.cpython-311.pyc
index 268c949..e468b20 100644
Binary files a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/__pycache__/rough_env_cfg.cpython-311.pyc and b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/__pycache__/rough_env_cfg.cpython-311.pyc differ
diff --git a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc
index 3781601..dfd1ec5 100644
Binary files a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc and b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/config/traquad/agents/__pycache__/rsl_rl_ppo_cfg.cpython-311.pyc differ
diff --git a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/velocity_env_traquad_cfg.py b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/velocity_env_traquad_cfg.py
index 006b92c..778a63b 100644
--- a/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/velocity_env_traquad_cfg.py
+++ b/ISAAC_NOGIT/source/isaaclab_tasks/isaaclab_tasks/manager_based/locomotion/velocity/velocity_env_traquad_cfg.py
@@ -95,7 +95,7 @@ class CommandsCfg:
         heading_control_stiffness=1.0,
         debug_vis=True,
         ranges=mdp.UniformVelocityCommandCfg.Ranges(
-            lin_vel_x=(-1.0, 1.0), ang_vel_z=(-1.0, 1.0)
+            lin_vel_x=(-1.0, 1.0), lin_vel_y=(0.0, 0.0), ang_vel_z=(-1.0, 1.0)
         ),
     )
 